README.md

```diff
@@ -189,6 +189,34 @@ loader Elasticsearch is on the same machine as TweetSets (e.g., in a small devel
         --conf spark.driver.host=$SPARK_DRIVER_HOST \
         tweetset_loader.py spark-create /dataset/path/to
 
+### Reloading an existing set with Apache Spark
+1. Start and connect to a loader container:
+
+        docker-compose -f loader.docker-compose.yml run --rm loader /bin/bash
+2. Invoke the loader:
+
+        spark-submit \
+        --jars elasticsearch-hadoop.jar \
+        --master spark://$SPARK_MASTER_HOST:7101 \
+        --py-files dist/TweetSets-2.0-py3.6.egg,dependencies.zip \
+        --conf spark.driver.bindAddress=0.0.0.0 \
+        --conf spark.driver.host=$SPARK_DRIVER_HOST \
+        tweetset_loader.py spark-reload dataset-id
```

Comment:
The spark-reload command also requires a /dataset/path/to argument. This line should be:
`tweetset_loader.py spark-reload dataset-id /dataset/path/to`

Reply:
@lwrubel path is now added.  Please review.